modules/scheduler/gke-cluster/main.tf

```diff
@@ -196,9 +238,12 @@ resource "google_container_node_pool" "system_node_pools" {
   provider = google-beta
   count    = var.system_node_pool_enabled ? 1 : 0
 
-  project = var.project_id
-  name    = var.system_node_pool_name
-  cluster = google_container_cluster.gke_cluster.self_link
+  project  = var.project_id
+  name     = var.system_node_pool_name
+  cluster  = var.is_a3_ultra ? google_container_cluster.gke_cluster.name : google_container_cluster.gke_cluster.self_link
+  version  = var.is_a3_ultra ? var.min_master_version : null
+  location = var.is_a3_ultra ? "us-east5-staginga" : null
```

Comment:
I am assuming this is only a temporary issue ? Also why can't we just use blueprint `var.zone` here ?

Reply:
Or are we creating a zonal GKE cluster for this test ?

Reply:
This is required by the sandbox environment.

We can use the `var.zone`. Will make the change

Reply:
I guess we are creating a multi-zonal cluster if passing a zone in the location : https://screenshot.googleplex.com/9CEVTFgbA9RyudG

So agreed, but if we want to keep testing this in future, we should probably add first class support for multi-zonal cluster.